kimminwoo/likelion-assignment-crud/src/main/java/com/kimminwoo/likelionassignmentcrud/student/domain/Student.java

```diff
@@ -0,0 +1,34 @@
+package com.kimminwoo.likelionassignmentcrud.student.domain;
+
+import com.kimminwoo.likelionassignmentcrud.assignment.domain.Assignment;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.ArrayList;
+
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Student {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "student_id")
+
+    private Long studentId;
+    private String name;
+    private String phoneNumber;
```

Comment:
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "student_id")
    private Long studentId;
    
    private String name;
    
    private String phoneNumber;
```

줄 바꿈과 코드 포맷팅도 개발의 중요한 부분입니다!

가독성을 높이기 위해 필드 간에 줄바꿈을 일정하게 해주는 것이 좋아요.
위 예시처럼 어노테이션이 붙은 필드와 일반 필드 사이에 구분되는 줄바꿈을 적용하면
코드를 읽는 사람이 더 쉽게 구조를 파악할 수 있습니다 😊

예를 들어, 엔티티에서 주요 식별자 필드(`@Id`가 붙은 부분)와 그 외 필드를 구분해주는 식으로 작성하면,
전체 클래스 구조가 훨씬 명확하게 보여요!